src/api/app/views/webui/package/binary.html.haml

```diff
@@ -0,0 +1,54 @@
+- @pagetitle = "Detailed Information About #{@filename}"
+- @metarobots = 'noindex' # files change too often
+- package_bread_crumb(link_to('Binaries', package_binaries_path(project: @project,
```

Comment:
We don't use it anymore

Reply:
I am aware of that:smiley: As stated above, this is just converting a view of the old webui to HAML:wink: